### DISCUSSION TITLE
Feat: Ability to change currency for model usage costs

### DISCUSSION THREAD

**Discussion by jannikstdl** [langfuse/langfuse#2586] [Ideas]
### Describe the feature or potential improvement

Hello,

i would like to be able to change the currency for the model usage for example in EUR.

**Comment by marcklingen** [comment]
thanks for adding this

**Reply by raj-triforce** [reply]
yes, i would like this feature where you can configure currency according to your country, but if you can at least add the currency for the major population who use AI, US(which is already there), Europe, China, Singapore, UK, India, Australian dollars and who ever  left

**Comment by antonays** [comment]
+1

**Comment by raoufamara** [comment]
+1
